File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_idxaltrn.cs
// EXPECTED: 88.25
// Reference emit for IDXALTRN — GnuCOBOL INDEXED ALTERNATE KEY read.
using System;
using System.Collections.Generic;
public static class IdxaltrnRef {
  public static void Main() {
    var rows = new Dictionary<string, double> {
      ["ALT00010"] = 12.50, ["ALT00088"] = 88.25,
    };
    Console.WriteLine(rows["ALT00088"].ToString("0.00"));
  }
}

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_idxaltrn.java
// EXPECTED: 88.25
// Reference emit for IDXALTRN — GnuCOBOL INDEXED ALTERNATE KEY read.
public final class IdxaltrnRef {
  public static void main(String[] args) {
    java.util.Map<String, Double> rows = java.util.Map.of(
        "ALT00010", 12.50, "ALT00088", 88.25);
    System.out.printf("%.2f%n", rows.get("ALT00088"));
  }
}
